packages/ui/src/kit/VButton/VButton.theme-default.scss

```diff
@@ -102,6 +102,10 @@
   color: var(--endpass-ui-color-primary-7);
 }
 
+.v-button.theme-default.skin-ghost-error .v-button-label {
```

Comment:
add for this case new stories in `VButton.stories.js`
and for the changes below too for visual checking that all is fine

Reply:
this is old case, see split mode - I just move it due to `no-descendant-specifity` rule

Reply:
About `ghost-error` - there is in story as well, just fix bug. In old version there are incorrect skin.